utils/mil_tools/nodes/online_bagger.py

```diff
@@ -54,10 +54,31 @@ def get_params(self):
         """
         Retrieve parameters from param server.
         """
+        self.dir = rospy.get_param('~bag_package_path', default=None)
```

Comment:
Just wondering if you still wanted to have this default to a dated folder under the directory specified in the BAG_DIR environment variable if it exists. You added a thumbs up to the comment, so I think you just missed it when making changes.